docs/user/config-file/v2.rst

```diff
@@ -328,9 +328,6 @@ You can use several interpreters and versions, from CPython, PyPy, Miniconda, an
   - ``3.9``
   - ``3.10``
   - ``3.11``
-  - ``pypy3.7``
-  - ``pypy3.8``
-  - ``pypy3.9``
```

Comment:
I do think it makes sense more as an option for builds.tools, but I'm pretty surprised nobody is using this. If it in fact isn't used by anyone, I'm fine removing it 👍